third-party/linshare/src/test/java/org/apache/james/linshare/client/LinshareAPITest.java

```diff
@@ -255,7 +255,8 @@ private File templateFile() throws Exception {
         return Files.createTempFile("linshare-api-test", ".temp").toFile();
     }
 
-    private List<Document> sharedDocs(List<ReceivedShare>...shares) {
+    @SafeVarargs
+    private final List<Document> sharedDocs(List<ReceivedShare>...shares) {
```

Comment:
We are missing a space after `...` no?

Reply:
not detected by checkstyle, but fixed anyway